Build on win

skupr-anaconda opened the pull request.

jjhelmus committed:
update to use compiler conda packages

jjhelmus committed:
include site.cfg file

jjhelmus committed:
include site.cfg file

jjhelmus committed:
multiple outputs

split openblas into multiple output packages:

libopenblas : The dynamic libraries
libopenblas-static : static libraries
openblas-devel : developement files (headers, config, etc)
openblas : metapackage of libopenblas and openblas-devel

jjhelmus committed:
multiple outputs

split openblas into multiple output packages:

libopenblas : The dynamic libraries
libopenblas-static : static libraries
openblas-devel : developement files (headers, config, etc)
openblas : metapackage of libopenblas and openblas-devel

jjhelmus committed:
add support for osx

jjhelmus committed:
add support for osx

jjhelmus committed:
add build requirements to libopenblas package

libopenblas should use the run_exports from the compilers, supply this by
adding these are build requirements to the sub-package.

jjhelmus committed:
add build requirements to libopenblas package

libopenblas should use the run_exports from the compilers, supply this by
adding these are build requirements to the sub-package.

jjhelmus committed:
explicit selection of files in outputs

Explicitly define the files which are included in each output.  Conda build
does not exclude files already present in the prefix so globs included too many
files.

Include BUILD_PREFIX in library search path to pick up needed libraries during
tests.

Bump the build number.

jjhelmus committed:
explicit selection of files in outputs

Explicitly define the files which are included in each output.  Conda build
does not exclude files already present in the prefix so globs included too many
files.

Include BUILD_PREFIX in library search path to pick up needed libraries during
tests.

Bump the build number.

msarahan committed:
add explicit make dep

msarahan committed:
add explicit make dep

msarahan committed:
add blas mutex package; run_exports for it

msarahan committed:
add blas mutex package; run_exports for it

msarahan committed:
add jom and cmake deps on win.  Fix blas metapackage build str

msarahan committed:
add jom and cmake deps on win.  Fix blas metapackage build str

msarahan committed:
add nomkl package; fix tests on cross

msarahan committed:
add nomkl package; fix tests on cross

msarahan committed:
whitespace cleanup

msarahan committed:
whitespace cleanup

msarahan committed:
add msarahan maintainer; change win skip

msarahan committed:
update; fix hardcoded fortran flags

msarahan committed:
Merge branch 'master' of github.com:AnacondaRecipes/openblas-feedstock

mingwandroid committed:
Try segfaults.patch (did not help)

mingwandroid committed:
Clean-up recipe

mingwandroid committed:
Debugging ..

mingwandroid committed:
Add verbosity to f_check

This is failing to find our fortran compiler somehow and we default to F77
but that's not ABI compatible with the C code that calls it.

mingwandroid committed:
Fix patch

mingwandroid committed:
Fix gfortran detection in the face of ctng compiler version strings

mingwandroid committed:
Revert "Debugging .."

This reverts commit 02fed4d687dc48dedf1095e8d2fd58c14df22a52.

mingwandroid committed:
Minor tidy ups

msarahan committed:
bump build number; add nomkl run deps

msarahan committed:
turn openmp off on mac, due to llvm/gcc mix concerns

mingwandroid committed:
Quieten some pointless warnings, run lapack tests, exit with failure if tests fail.

mingwandroid committed:
Update to 0.3.3

msarahan committed:
update pins to be less constrained; reflect good ABI compatibility

msarahan committed:
set CPPFLAGS for mac so that tests run

msarahan committed:
set CFLAGS and FFLAGS during test stages (fix mac fail w/missing rpath)

msarahan committed:
turn TLS off (prematurely enabled by default)

msarahan committed:
disable openmp (GNU openmp not fork safe; causing hangs)

mingwandroid committed:
Do not use clang-cl or flang

Commit:
0.3.6

isuruf committed:
Set SONAME for osx

isuruf committed:
Update meta.yaml

isuruf commented:
cc @soapy1

isuruf commented:
ping

isuruf commented:
ping again

isuruf commented:
@kalefranz, @mingwandroid, @msarahan, @jjhelmus  can one of you please merge this? I'm trying to make defaults and conda-forge compatible and this has been sitting here for more than a month.

jjhelmus committed:
Merge pull request #3 from isuruf/patch-1

Set SONAME for osx

isuruf commented:
Thanks. I've also sent this upstream and the next release should have a install_name without the full version in it.

jjhelmus commented:
Sorry for overlooking this @isuruf.  I'll get this built.  I want to get the [SkyLake-X patch]( (https://github.com/conda-forge/openblas-feedstock/blob/master/recipe/gh2111.patch) merged as well as this has been causing some issues with NumPy and Numba.

jjhelmus committed:
disable SkyLakeX DGEMMIxCOPY kernels

See https://github.com/xianyi/OpenBLAS/pull/2111

msarahan committed:
move nomkl track_feature to blas metapackage

this is already done via hotfix.  Just making the recipe here consistent.

jjhelmus committed:
add netlib license

closes #5

chenghlee committed:
Update to OpenBLAS 0.3.10

chenghlee committed:
Rewrite build.sh to collect build options into an array

On some platforms, some of these options need to also be passed to
`make lapack-test` and `make install` for them to function correctly.
Collecting these options into an array reduces the changes of a
copy-paste error that could result in a (silently) broken build.

chenghlee committed:
Update recipe for linux-s390x

katietz committed:
add s390x support

chenghlee committed:
Merge branch 'linux-on-z'

ilanschnell committed:
avoid ulimit call on linux-aarch64

chenghlee committed:
Update version to 0.3.13

Perform minor clean up of build script and drop obsolete patch

chenghlee committed:
Fix AVX-512 intrinsics header for GCC 7.3

Ugly hack so we can build OpenBLAS using the older gcc release (7.3.0)
that's the linux-64 compiler for default; without this, compilation of
Skylake X kernels (and possibly others) will fail with `incompatible
type for argument 1 of '_mm512_abs_pd'` errors.

Using this hack because updating the compilers for the "defaults"
channel is much more complicated than we had anticipated.

chenghlee committed:
Merge branch 'openblas-v0.3.13'

katietz committed:
set ppc64le using gcc 8.2.0

skupr-anaconda committed:
Fix license, add license_family

skupr-anaconda committed:
Add patch on unix

skupr-anaconda committed:
Bump build number to 2

skupr-anaconda committed:
Merge pull request #8 from AnacondaRecipes/ppc64le-8.2.0gcc

ppc64le 8.2.0gcc

katietz committed:
add osx-arm64 specific changes

katietz committed:
update to 0.3.17

katietz committed:
add patch dependency

katietz committed:
adjust license name

katietz committed:
add a hack for conda-build bug related to --skip-existing

katietz committed:
disable ulimit call on linux-64 due prefect's docker image limitation

katietz committed:
add SDK library path to FFLAGS for fortran

katietz committed:
add c++ dependency

katietz committed:
replace -L by -isysroot for fflags of compiler

katietz committed:
fix redundant regeneration of nomkl, and resolve missing headers in developer package

katietz committed:
add some comment about nan patch

katietz committed:
Merge pull request #9 from AnacondaRecipes/update-0.3.17

update to 0.3.17

antkneed committed:
Release v0.3.18 (#10)

* Update to v0.3.18.

* Remove upstreamed patch.

* Upload to c3i_test2 for testing.

* urls: http -> https

* Revert "Upload to c3i_test2 for testing."

This reverts commit 8a9e1798009490db8e9bfd5f21c030689a928d0c.

ScottBailey committed:
Update to v0.3.20 and use gcc11 on Linux. (#12)

* Update to v0.3.20 and use gcc11 on Linux.
* Added license_family

tobijk committed:
openblas-0.3.20 b1

tobijk committed:
Merge pull request #13 from AnacondaRecipes/openblas-0.3.20-b1

openblas-0.3.20 b1

tobijk committed:
Retrigger build to generate missing osx-arm64 artifact

tobijk committed:
Merge pull request #14 from AnacondaRecipes/openblas-0.3.20-b1-t1

Retrigger build to generate missing osx-arm64 artifact

cbouss committed:
update to 0.3.21

cbouss committed:
Revert "update to 0.3.21"

This reverts commit d9246377204a68b206b60f179d3e7f67af82ae4d.

cbouss committed:
update to 0.3.21

cbouss committed:
openblas removed the build system requirements for PERL

cbouss committed:
osx-arm64 workaround gfortran11.2

cbouss committed:
update about section

cbouss committed:
remove abs.yaml

cbouss committed:
add back abs.yaml

cbouss committed:
Merge pull request #15 from AnacondaRecipes/openblas-0.3.21

Openblas-0.3.21

skupr-anaconda committed:
Build on win

skupr-anaconda committed:
Build on win
